### ISSUE TITLE
Ban Appeal

### ISSUE THREAD

**Issue by Phattty** [ObliqueNET/Stratos#290]
I was banned for a day earlier for a justified reason, as I had sent a racist piece of mail to my friend. I realized now it was a bad choice, and was fine with the one day ban. However at the time of the ban I was unsure of why, and went on terra to ask. However later I tried to log back onto stratos to see how long I was banned for, and found it was now permanent for causing trouble on Terra. Although I did repeatedly ask why I was banned in chat on Terra, I did not actually pvp anyone without consent unlike my friend who was also banned from Terra, and did not do anything to interrupt other players enjoyment of the game. I also stopped later after I realized I was wrong, and said that I would accept the one day ban in public chat. I'm sorry for my original actions, but feel that it should be reduced to its original one day sentence.

              Sincerely, Phattty

**Comment by Grassip** [comment]
You weren't permanently banned because of what you did on Terra, but because of what you did on Stratos. The reason for ban was because you were being racist towards a player. I know that you went to Terra and asked the staff repeatedly why you were banned and continued to be disruptive to the point that the staff there had to ask you to cut it out, so I mentioned it in the reasoning. We understand that you stopped trying to talk about it on Terra, but that's not the only reason you were perm banned. The ban was originally supposed to be permanent because of your racism. 

Also, you _were_ warned quite a few times, once you were muted for it and warned you did it another time by giving Chris this:
![image](https://user-images.githubusercontent.com/47408346/60699017-a7077b80-9eb7-11e9-8aa7-d1bc966ff829.png)

So after the many times you were warned about it in chat, you continued to go as far as to give him a letter with the name that Chris had already told you multiple times before was offensive. This shows that you had no regards for the way he felt and had little to no self control in the situation, and therefore you will not be unbanned.

**Comment by Phattty** [comment]
Although I see why he did not want to be called this name, the letter was sent before he told me to stop calling him this name. He did not in fact warn me multiple times, but said in chat once to not call him that after I gave him the letter. After I was told this I stopped, and muted which was understanble. I didn’t even say another word in chat until I was given my original one day ban, which occurred while I was muted. Although I see how this could have offended him, and feel sorry about it, I stopped after he warned me, and he did not mention it before I sent the letter. As a result, I think the ban should be reduced back to its original sentence.
